infracost-usage-example.yml

```diff
@@ -703,6 +703,9 @@ resource_usage:
   azurerm_private_dns_txt_record.my_txt_record:
     monthly_queries: 1500000000  # Monthly number of DNS queries
 
+  azurerm_redis_cache.my_redis_cache:
+    redis_nodes: 5 # Override the number of nodes. 
```

Comment:
It might be good enough to remove this as the user can only increase node count via terraform and the params that are being used (replicas_per_master and shard_count)? If our calculation is wrong, we can fix it when expert users report that?